gh-issue-create: add page (#6039)
tldr-pages/tldr: pages/common/gh-issue-create.md

# gh issue

> > Create issues in GitHub from the command-line.

Reviewer: ```suggestion
> Create issues for GitHub from the command-line.
```

Reviewer 2: Neither `in` nor `for` work well here.  `on` sounds the best, but I'd rearrange it so as to not have to use the preposition:
```suggestion
> Create GitHub issues on a repository from the command-line.
```

> More information: <https://cli.github.com/manual/gh_issue_create>.

- Create a new issue for the current repository interactively:

`gh issue create`

> - Create a new issue with a bug label interactively:

Reviewer 3: Perhaps this will read better?
```suggestion
- Create a new issue with the `bug` label interactively:
```

`gh issue create --label "{{bug}}"`

> - Create a new issue interactively and assign it to 2 users:
> 
> `gh issue create --assignee {{user1,user2}}`

Reviewer 4: How about:
```suggestion
- Create a new issue interactively and assign it to the specified users:

`gh issue create --assignee {{user1,user2,...}}`
```

> - Create a new issue with a title, body and assigned to the current user:

Reviewer 4: ```suggestion
- Create a new issue with a title, body and assign it to the current user:
```
I think that sounds better.

> `gh issue create --title "{{title}}" --body "{{body}}" --assignee "{{@me}}"`

Reviewer 4: Should we maybe remove `--assignee` here, since it was already introduced in the previous example? Also, is this not interactive anymore?

Author: If you set the required params in the CLI arguments (`title` and `body`) the command is not interacive. Also, I'm trying to say that `@me` is a wildcard for the current user of the comamnd, is not the same than `user1`.

- Create a new issue interactively reading the body text from a file:

`gh issue create --body-file {{path/to/file}}`

> - Create a new issue in the web browser:

Reviewer 3: Perhaps this will read better?
```suggestion
- Create a new issue in the default web browser:
```

`gh issue create --web`

> - Display help:

Reviewer 3: Perhaps this will read better?
```suggestion
- Display the help:
```

Reviewer: I dislike this change. Display help actually reads better IMO, print would be better as well
